sdk/communication/Azure.Communication.MediaComposition/src/MediaCompositionClient.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Is this a manual convenience layer that you wrote? @thdinizm

Reply:
I think we are not planning to invest too much into writing our own convenience layer. We could just use `MediaCompositionRestClient` directly right now in the samples. 

If we do want to have a manually written convenience layer, there are a few guidelines to follow such as having an async and sync version of all the functions, thinking about usability etc. It will take more effort to do it properly.

Reply:
It seemed impossible to me to not have a convenience layer on C#, I'd need to manually change some of the generated code otherwise. That's why I chose to build the simplest layer I could for now..